xml/System.Data/DataReaderExtensions.xml

```diff
@@ -38,11 +38,13 @@
         <Parameter Name="name" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="reader">To be added.</param>
-        <param name="name">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="reader">The <see cref="T:System.Data.Common.DbDataReader" /> to get the column value from.</param>
+        <param name="name">The name of the column.</param>
+        <summary>Gets the value of the specified column as a Boolean.</summary>
+        <returns>The value of the specified column.</returns>
         <remarks>To be added.</remarks>
```

Comment:
Remove all empty remarks?

Reply:
Ok. Thanks for clarifying! I will do a scan when this is closer to finished to make sure I didn't delete anything accidentally.